src/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,44 @@
+import { LinkWithQuery } from "@components/LinkWithQuery";
+
+type TBreadcrumbLink = {
+  label: string | React.ReactNode;
+  href?: string;
+  last?: boolean;
+};
+
+type TProps = {
+  category?: TBreadcrumbLink;
+  family?: TBreadcrumbLink;
+  geography?: TBreadcrumbLink;
+  label: string | React.ReactNode;
+};
+
+const BreadCrumb = ({ last = false, label, href = null }: TBreadcrumbLink) => {
+  return (
+    <>
+      <li>
+        {href ? (
+          <LinkWithQuery className="underline" href={href}>
```

Comment:
I just want to check that this code in `LinkWithQuery` is not going to cause any issue with breadcrumbs (can't quite work it out myself)
```
  // Remove specific slug parameters from the query string
  delete router.query["docId"];
  delete router.query["id"];
  delete router.query["geographyId"];

```